packages/engine/Cargo.toml

```diff
@@ -46,9 +46,8 @@ tracing-appender = "0.2.0"
 tracing-error = "0.2.0"
 tracing-subscriber = { version = "0.3.7", features = ["env-filter", "json"] }
 tracing-texray = { version = "0.1.2", optional = true }
-# Only temporary, use the crates.io release when ArrayBuffer::new_backing_store_from_ptr is available
-# https://github.com/denoland/rusty_v8/pull/926
-v8 = { git = "https://github.com/hashdeps/rusty_v8", rev = "0bf4d26961697591baf54bc4ceb0d15d3f195f86" }
+# Only temporary, use the next crates.io release
```

Comment:
Can we add the current version to the comment so it's easier to check, we can use the crates.io version?

Reply:
Added the version https://github.com/hashintel/hash/pull/487/commits/de20f7a84176e91c8f8b108da4681a676095d289